### PR TITLE
Feature assignment17

### PR DESCRIPTION
**Pull request by dheerajsrinath**
Create an abstract class WillDoEverthing in that create one method sum(int a), Call that sum()
method from Class Yes1 and do addition of three numbers.
Note - Use Interface concept

### CODE REVIEW

File: src/main/java/com/assignment/Yes1.java [hunk 1]
```diff
@@ -0,0 +1,12 @@
+package com.assignment;
+
+interface WillDoEverything {
+    public int sum(int a);
+}
+
+public class Yes1 implements WillDoEverything {
```
Comment (PranavHanwante) [inline src/main/java/com/assignment/Yes1.java]: give a meaningful name for a class name, Yes I know in the assignment its given Yes1 but change it

File: src/main/java/com/assignment/Yes1.java [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+
+    public int sum(int a) {
```
Comment (PranavHanwante) [inline src/main/java/com/assignment/Yes1.java]: change method namr to get

File: src/test/java/com/assignmentTest.java [hunk 3]
```diff
@@ -0,0 +1,20 @@
+package com;
+
+import com.assignment.Yes1;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public class assignmentTest {
```
Comment (PranavHanwante) [inline src/test/java/com/assignmentTest.java]: class name starting letter should be alphabetic

File: src/test/java/com/assignmentTest.java [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+    private Logger logger = LoggerFactory.getLogger(this.getClass());
+
+    @Test
+    public void testInterface() {
+        int intToAdd = 1;
+        Yes1 yes1 = new Yes1();
+        logger.info(Integer.toString(yes1.sum(intToAdd)));
```
Comment (PranavHanwante) [inline src/test/java/com/assignmentTest.java]: intent

File: src/test/java/com/assignmentTest.java [hunk 5]
```diff
@@ -0,0 +1,20 @@
...
+        Assertions.assertTrue(yes1.sum(intToAdd) == (1 + 1 + 1),
```
Comment (PranavHanwante) [inline src/test/java/com/assignmentTest.java]: instead of 1+1+1, you can directly use 3

File: src/test/java/com/assignmentTest.java [hunk 6]
```diff
@@ -0,0 +1,20 @@
...
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public class assignmentTest {
+    private Logger logger = LoggerFactory.getLogger(this.getClass());
+
+    @Test
+    public void testInterface() {
```
Comment (PranavHanwante) [inline src/test/java/com/assignmentTest.java]: change name

**Review by PranavHanwante — APPROVED** [review]

**Review by akhil7000 — APPROVED** [review]

**Pull request merged**